packages/web/src/scss/components/Modal/README.md

```diff
@@ -274,15 +274,20 @@ limit.
 
 You can set a custom preferred height of ModalDialog using a custom property:
 
-- `--modal-preferred-height-mobile` for mobile screens, and
-- `--modal-preferred-height-tablet` for tablet screens and up.
+- `--modal-height` for mobile screens and up,
```

Comment:
shouldn't all these variables be `--modal-dialog-*`? As it sets height of the dialog and you place it on the dialog etc.

Reply:
You are right, I renamed them all to `--modal-dialog-*`.